apps/api/src/app/events/usecases/send-message/send-message-chat.usecase.ts

```diff
@@ -115,7 +115,14 @@ export class SendMessageChat extends SendMessageBase {
     }
 
     for (const channel of chatChannels) {
-      await this.sendChannelMessage(command, channel, notification, chatChannel, content);
+      try {
+        await this.sendChannelMessage(command, channel, notification, chatChannel, content);
+      } catch (e) {
+        /*
+         * Do nothing, one chat channel failed, perhaps another one succeeds
+         * The failed message has been created
+         */
```

Comment:
@ryshoooo maybe we can add here an execution detail for failed delivery? Or maybe throw if all channels have failed? Because otherwise, the trigger will appear as delivered, when in reality it wasn't

Reply:
Also let's log the ones that fail. You can do it with:
```
const LOG_CONTEXT = 'SendMessageChat'; // Make this available in the top of the file.
...
Logger.error(`Sending chat message to the chat channel ${channel}`, LOG_CONTEXT);
```
You can import the `Logger` from `@nestjs/common` package.

Reply:
Updated :)